elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -0,0 +1,414 @@
+import prairielearn as pl
+import lxml.html
+import random
+import chevron
+import base64
+import os
+import json
+
+PL_ANSWER_DEFAULT_CORRECTNESS = True
+PL_ANSWER_DEFAULT_INDENT = '-1'
+CHECK_PL_ANSWER_INDENTION_DEFAULT = False
+SHUFFLE_MCQ_OPTIONS_DEFAULT = False
+DEFAULT_GRADING_MODE = 'ordered'
+DEFAULT_SOLUTION_PLACEMENT = 'right'
+
+
+
+def render_html_color(score):
+    # used to render the correct color depending on student score
+    if score == 0:
+        return 'badge-danger'
+    elif score == 1.0:
+        return 'badge-success'
+    else:
+        return 'badge-warning'
+
+
+def get_all_answer(submitted_blocks, block_indents, leading_code, trailing_code):
+    if len(submitted_blocks) == 0:
+        return ''
+    answer_code = ''
+    for index, answer in enumerate(submitted_blocks):
+        indent = int(block_indents[index])
+        answer_code += ('    ' * indent) + answer + '\n'
+    answer_code = leading_code + '\n' + answer_code + trailing_code + '\n'
+    return answer_code
+
+
+def prepare(element_html, data):
+    element = lxml.html.fragment_fromstring(element_html)
+
+    pl.check_attribs(element,
+                     required_attribs=['answers-name'],
+                     optional_attribs=['shuffle-source-blocks',
+                                       'grading-method',
+                                       'check-indentation',
+                                       'source-header',
+                                       'solution-header',
+                                       'file-name',
+                                       'leading-code',
+                                       'trailing-code',
+                                       'solution-placement',
+                                       'max-incorrect',
+                                       'min-incorrect',
+                                       'weight'])
+
+    answer_name = pl.get_string_attrib(element, 'answers-name')
+
+    mcq_options = []
+    html_ordering = []
+    correct_answers = []
+    correct_answers_indent = []
+    correct_answers_ranking = []
+    incorrect_answers = []
+
+    for html_tags in element:  # iterate through the tags inside pl-order-blocks, should be <pl-answer> tags
+        if html_tags.tag == 'pl-answer':
+            # correct attribute is not strictly required, as the attribute is irrelevant for autograded questions
+            pl.check_attribs(html_tags, required_attribs=[], optional_attribs=['correct', 'ranking', 'indent'])
+
+            isCorrect = pl.get_boolean_attrib(html_tags, 'correct', PL_ANSWER_DEFAULT_CORRECTNESS)
+            answerIndent = pl.get_string_attrib(html_tags, 'indent', PL_ANSWER_DEFAULT_INDENT)  # get answer indent, and default to -1 (indent level ignored)
```

Comment:
There still seem to be a lot of variable names using camelCase, contrary to PEP8. Would you mind fixing all the variables in the python code to conform to PEP8 style (e.g., snake_case for python variables).

Reply:
fixed